cmd/download.go

```diff
@@ -39,13 +39,12 @@ func DownloadCmd() *cobra.Command {
 
 	cmd.Flags().StringVar(&host, "host", "", "stencil host address eg: localhost:8000")
 	cmd.MarkFlagRequired("host")
-	cmd.Flags().StringVar(&req.Namespace, "namespace", "", "provide namespace/group or entity name")
+	cmd.Flags().StringVar(&req.NamespaceId, "namespace", "", "provide namespace/group or entity name")
```

Comment:
All our APIs would be referring namespace and schema by it's name. So added it as string. In DB layer we still keep string as unique ID for namespace since it's unique across stencil. For schema we do have bigint id, we use combination of `namespace+schema_name` to get schema ID at DB layer